### DISCUSSION TITLE
The reason of NOT outputing non-mandatory arguments in Spectre.Console.Cli help output

### DISCUSSION THREAD

**Discussion by mickvikt** [spectreconsole/spectre.console#512] [General]
Hello, I just have a question/suggestion about the output of help information in Spectre.Console.Cli.
I've been creating a small CLI utility and noticed that an argument with a default value isn't being output to the screen when the program is run without arguments and help screen is displayed. I see from source, that this is done on purpose. On the other hand, when I run the command with a -h switch, it IS being output.

Now thinking from the usability perspective, you have to specifically execute the program with the help switch to get default arguments displayed. Sometimes you don't even now that there are any default arguments and you probably won't even try to query for help information.

I think it is a common practice/usage models to display non-mandatory arguments when the program is run without any arguments.

What is your reasoning on this?
Thank you.

**Comment by FrankRay78** [comment]
Better late than never - showing default values in help is now supported, see: https://github.com/spectreconsole/spectre.console/pull/1032
